modules/bullet/rigid_body_bullet.h

```diff
@@ -202,11 +202,11 @@ class RigidBodyBullet : public RigidCollisionObjectBullet {
 	bool omit_forces_integration = false;
 	bool can_integrate_forces = false;
 
-	Vector<CollisionData> collisions;
-	Vector<RigidBodyBullet *> collision_traces_1;
-	Vector<RigidBodyBullet *> collision_traces_2;
-	Vector<RigidBodyBullet *> *prev_collision_traces;
-	Vector<RigidBodyBullet *> *curr_collision_traces;
+	std::vector<CollisionData> collisions;
+	std::vector<RigidBodyBullet *> collision_traces_1;
+	std::vector<RigidBodyBullet *> collision_traces_2;
+	std::vector<RigidBodyBullet *> *prev_collision_traces;
+	std::vector<RigidBodyBullet *> *curr_collision_traces;
 
```

Comment:
I thought we weren't supposed to use `std::vector`.

Reply:
Removed `std::vector` in favour of the recently added `LocalVector`.